services/QuillLMS/app/queries/snapshots/options_query.rb

```diff
@@ -2,10 +2,13 @@
 
 module Snapshots
   class OptionsQuery < ::QuillBigQuery::Query
-    attr_accessor :admin_id
+    attr_accessor :admin_id, :school_ids, :grades, :teacher_ids
 
-    def initialize(admin_id, options = {})
+    def initialize(admin_id, school_ids = nil, grades = nil, teacher_ids = nil, options: {})
```

Comment:
Now that the method signature is getting longer, why not make `school_ids`, `grades`, and `teacher_ids` keyword params as well?